ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,151 @@
+[[!meta title="Renaming Accounts"]]
+
+Sometimes it's desirable to rename an OCF account.
```

Comment:
I definitely agree, putting the content of the "Closing thoughts" section at the top here seems like a better place to put that, since it will then be read first.